src/generateDiff.test.js

```diff
@@ -0,0 +1,325 @@
+const _outdent = require('outdent');
+
+const outdent = _outdent({ trimTrailingNewline: false });
+const { generateDiff } = require('./generateDiff');
+
+const testCases = [
+  {
+    description: 'should return an empty string when comparing empty files',
+    before: '\n',
+    after: '\n',
```

Comment:
Should these be actually empty (i.e. `''`) or is this as intended?

Reply:
This was intended - I was considering an "empty file" to be just a newline, as this is what an empty text file looks like on POSIX-like systems. All text files should end in newlines according to POSIX.

This just made for a nicer base test case, as the lack of newline notices really clutter things. Those are covered in other tests.